internal/authtoken/query.go

```diff
@@ -10,31 +10,31 @@ select reltuples::bigint as estimate from pg_class where oid in ('auth_token'::r
 	listAuthTokensTemplate = `
 with auth_tokens as (
      select public_id,
+            scope_id,
             auth_account_id,
             create_time,
             update_time,
             approximate_last_access_time,
             expiration_time,
             status
        from auth_token
+      where scope_id in @scope_ids
    order by create_time desc, public_id desc
       limit %d
 ),
 auth_accounts as (
     select public_id,
            auth_method_id,
-           scope_id,
            iam_user_id,
            iam_user_scope_id
       from auth_account
-     where %s
-       and public_id in (select auth_account_id from auth_tokens)
```

Comment:
This where clause was what concerned me with the old logic. It could lead to auth tokens being excluded after the limit had been applied, which will mess with the pagination logic. Lifting the scope_id into the first CTE applies the limit correctly _after_ excluding tokens from other scopes.